### PR TITLE
[11.0][MIG] account_tax_balance

### PR DESCRIPTION
**Pull request by pedrobaeza**
Standard migration

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @jbeficent @sbidoul

File: account_tax_balance/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Giovanni Capalbo <giovanni@therp.nl>
+* Tecnativa - Antonio Espinosa
+* Tecnativa - Pedro M. Baeza
```
Comment (rafaelbn) [inline account_tax_balance/README.rst:47]: Standard format is "Lorenzo Battistini <lorenzo.battistini@agilebg.com>"

Isn't it?

File: account_tax_balance/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016 ACSONE SA/NV - Stéphane Bidoul
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Tax Balance",
+    "summary": "Compute tax balances based on date range",
+    "version": "11.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://www.agilebg.com/",
```
Comment (rafaelbn) [inline account_tax_balance/__manifest__.py:12]: Should we change as discussed in the list to the OCA webpage?

File: account_tax_balance/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply account_tax_balance/README.rst:47]: I prefer not to put here email

File: account_tax_balance/__manifest__.py (hunk 2 above)
Comment (pedrobaeza) [reply account_tax_balance/__manifest__.py:12]: Well, it can be, but it's not a priority

**Review by oihane — APPROVED** [review]
LGTM only checked code

**Review by astirpe — APPROVED** [review]
LGTM (code review)

**Comment by pedrobaeza** [comment]
Merged as it's a dependency for a lot of modules of l10n-spain and there are enough approvals.

**Comment by levkar** [comment]
Taxes whose amount_type is group will always have zero balances. Would it be more convenient to hide/filter them?

**Comment by pedrobaeza** [comment]
Well, that's depend on how it's defined the subtaxes. I prefer to show both parent and children, but it can be a configuration.

**Pull request merged**